package.json

```diff
@@ -6,10 +6,18 @@
   "engines": {
     "node": ">=6.13.0"
   },
-  "scripts": {},
-  "dependencies": {},
+  "scripts": {
```

Comment:
I recommend alphabetizing your key-value pairs unless you have a compelling reason to do so otherwise.